dali/python/bundle-wheel.sh

```diff
@@ -121,6 +121,7 @@ DEPS_LIST=(
     "${DEPS_PATH}/lib/libopenjp2.so.7"
     "${DEPS_PATH}/lib/libzstd.so.1"
     "${DEPS_PATH}/lib/libz.so.1"
+    "${DEPS_PATH}/lib/libcfitsio.so.9"
```

Comment:
👍

Reply:
Ups, I didn't noticed the comment and forced pushed version from to commits behind. Now the line looks as follows (after changing build script to cmake version number changed): 
`    "${DEPS_PATH}/lib/libcfitsio.so.4"`